packages/app/src/systems/Accounts/hooks/useAccountConnectionEth.tsx

```diff
@@ -13,10 +21,13 @@ export function useAccountConnectionEth() {
     handlers: {
       connect,
       disconnect,
-      openAccountConnectionEth: store.openAccountConnectionEth,
       closeDialog: store.closeOverlay,
```

Comment:
can we remove this as well considering the eth connection dialog is now controlled by connect kit?

Reply:
yeah makes sense !